docs/root/start/sandboxes/double-proxy.rst

```diff
@@ -0,0 +1,174 @@
+.. _install_sandboxes_double_proxy:
+
+Double proxy (with encryption)
+==============================
+
+This sandbox demonstrates a basic "double proxy" configuration, in which a simple Flask app
+connects to a PostgreSQL database, with two Envoy proxies in between.
+
+``Envoy (front)`` -> ``Flask`` -> ``Envoy (postgres-front)`` -> ``Envoy (postgres-back)`` -> ``PostgreSQL``
+
+This type of setup is common in a service mesh where Envoy acts as a "sidecar" between individual services.
+
+It can also be useful as a way of providing access for application servers to upstream services or
+databases that may be in a different location or subnet, outside of a service mesh or sidecar-based setup.
+
+Another common use case is with Envoy configured to provide "Points of presence" at the edges of the cloud,
+and to relay requests to upstream servers and services.
+
+This example encrypts the transmission of data between the two middle proxies, using ``mTLS``. The proxies
+also validate and authenticate each others certificates.
+
+This can be useful if the proxies are physically separated or transmit data over untrusted networks.
+
+In order to  use the sandbox you will first need to generate the necessary SSL keys and certificates.
+
+This example walks through creating a certificate authority, and using it to create a domain key and sign
+certificates for the proxies.
```

Comment:
i guess here is where we want to say something about SNI

i just need to figure what exactly